packages/fxa-payments-server/src/routes/Home.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
```

Comment:
If anything, this page should be replaced with a static "you're in the wrong place" or a redirect back to FxA settings. The "Manage" button on FxA settings links to `/subscriptions`. For individual products, there's a route pattern of `/products/:productId`

Reply:
I think we should either make the settings page link to `/products/:ourOnlyProduct`, or link it to `/products` and have a default `productId` in the config. (This could all be a followup to this PR)

Reply:
Well, the /products route is for a new signup. /subscriptions is for managing existing subscriptions